pyproject.toml

```diff
@@ -27,24 +26,43 @@ packages = [
     { include = "humanloop", from = "src"}
 ]
 
+
 [project.urls]
 Repository = 'https://github.com/humanloop/humanloop-python'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 httpx = ">=0.21.2"
 httpx-sse = "0.4.0"
 pydantic = ">= 1.9.2"
 pydantic-core = "^2.18.2"
 typing_extensions = ">= 4.0.0"
+parse = "^1.20.2"
```

Comment:
is this file autogenerated?

Reply:
Yes - the added imports are introduced in this PR which will modify the pyproject.toml
https://github.com/humanloop/humanloop-docs/pull/147